client/src/js/jobs/components/Item/__tests__/Item.test.js

```diff
@@ -7,22 +7,36 @@ describe("<JobItem />", () => {
         props = {
             id: "foo",
             progress: 1,
-            state: "bar",
-            task: "baz",
+            task: "build_index",
             created_at: "Foo",
             user: {
-                id: "Bar"
+                id: "bob"
             },
             canCancel: true,
             canRemove: true,
-            handleCancel: "Baz",
-            handleRemove: "foe"
+            onCancel: jest.fn(),
+            onRemove: jest.fn()
         };
     });
-    it("should render", () => {
+
+    it.each(["waiting", "running", "cancelled", "error", "complete"])("should render when [state=%p]", state => {
+        props.state = state;
         const wrapper = shallow(<JobItem {...props} />);
         expect(wrapper).toMatchSnapshot();
     });
+
+    it.each([[true, true], [true, false], [false, true], [false, false]])(
+        "should render when [canCancel=%p] and [canRemove=%p]",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace `········"should·render·when·[canCancel=%p]·and·[canRemove=%p]",⏎·······` with `····])("should·render·when·[canCancel=%p]·and·[canRemove=%p]",`](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4918216)